generate_em_snapshot.m

```diff
@@ -0,0 +1,27 @@
+function img=generate_em_snapshot(em_data, start_time, duration, sx, sy)
+% img = generate_em_snapshot(em_data, from, win, sx=304, sy=240)
+%
+% Makes a snapshot of the gray levels contained in an em_data structure.
+% This snapshot consists of the last gray level value contained in em_data
+% for each pixel from 'start_time' and for 'duration' (all given in uS)
+%
+% 'sx' and 'sy' represent the size of the generated image and defaults to
+% the size of the sensor when non-specified
```

Comment:
Default to the size of the GEN1 sensor when non-specified. https://www.prophesee.ai/2020/01/24/prophesee-gen1-automotive-detection-dataset/ (or https://www.prophesee.ai/2020/01/24/prophesee-gen1-automotive-detection-dataset/)

Reply:
fixed!